### PR TITLE
WIP: Update recipe to version 34.1.1

### PR DESCRIPTION
**Pull request by jjhelmus**

### CODE REVIEW

**Comment by jjhelmus** [comment]
setuptools 34.x.x removed the vendored dependencies which requires that they be installed separately.  With 34.1.1 this can be done without pip.  There is still a bit of an issue with circular dependencies as `six`, `appsdir` and `packaging` require `setuptools` as a build requirement, but none of them require `setuptools` as a run time requirement.

**Comment by jakirkham** [comment]
Should we build the last release of 33 first? Any other actions we need to take with the dependencies?

**Comment by jjhelmus** [comment]
> Should we build the last release of 33 first?

I have no opinion on if a 33.1.1 release is needed.  The [changes](https://github.com/pypa/setuptools/blob/master/CHANGES.rst#v3311) for that release do not seem too major to warrent a seperate issue.



> Any other actions we need to take with the dependencies?

I do not believe so.  Since we already have `setuptools` conda packages and its dependencies, there should not be an issue with continuing to build new versions.  Creating a `setuptools` package for a new version of Python or a new platform will require some work but I think that can be dealt with when we get to that point.

**Comment by jakirkham** [comment]
Side note: Issue ( https://github.com/pypa/setuptools/issues/980 ) looks to be of interest.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -20,11 +21,17 @@ build:
 requirements:
   build:
     - python
+    - appdirs
     - certifi
+    - packaging
+    - six
```
Comment (jakirkham) [inline recipe/meta.yaml]: I guess we can't avoid having these as build dependencies too, right?

**Comment by jakirkham** [comment]
Merged with `master` to resolve conflicts.

**Comment by jakirkham** [comment]
Would really appreciate if some people took a look at and chimed in on issue ( https://github.com/pypa/setuptools/issues/980 ). The future of `setuptools` is under discussion as well as how one has to package it.

cc @conda-forge/core @conda-forge/setuptools @mingwandroid

**Comment by mingwandroid** [comment]
Thanks for the info @jakirkham, my only comment here is to say that being able to bootstrap an entire distribution is extremely important and that when dependencies end up cyclic that is no longer possible. Is my reading of the situation close to the mark?

If it is of any use as a data-point, I have successfully bootstrapped a cross-compiled Miniconda-like distribution based on `conda-forge` recipes recently with `setuptools 33.1.1`.

If there's anything I can do to help this situation the please let me know. Using pre-built binaries is not possible when bootstrapping for a new platform.

**Comment by mwcraig** [comment]
Sorry, but I really don't have anything useful to contribute (read the issue you referenced, at least until my eyes glazed over).

**Comment by patricksnape** [comment]
Yikes, that's a mess. Not sure what I can do but happy to help.

**Comment by nicoddemus** [comment]
Superseded by #66

**Pull request closed without merging**